demo/index.html

```diff
@@ -88,6 +88,54 @@ <h1>Shadow DOM examples</h1>
           <auro-input disabled value="disabled" type="email" label="First name" helptext="Enter your first name" required id="disabledEmailTest01"></auro-input>
         </template>
       </demo-snippet>
+
+      <demo-snippet>
+        <template>
+          <auro-input id="format-ccWithIcon" format='{"creditCard": true, "icon": true}' icon="true" label="Credit Card Number with Icon" required></auro-input>
+        </template>
+      </demo-snippet>
+
+      <demo-snippet>
+        <template>
+          <auro-input id="format-cc" format='{"creditCard": true, "icon": true}' label="Credit Card Number without Icon" required></auro-input>
+        </template>
+      </demo-snippet>
+
+      <demo-snippet>
+        <template>
+          <auro-input id="format-datelong" format='{"date": true, "delimiter": "-", "datePattern": ["Y", "m", "d"]}' label="Date YYYY-MM-DD"></auro-input>
```

Comment:
This is an interesting bonus, but for the final demo we will not promote support to usage at this time until we have completed other work that is in design.